Doesn't work with PyCharm 2017.3 EAP

EAP builds are available here: https://www.jetbrains.com/pycharm/nextversion/

Got following output running tests under debugger with `pytest-pycharm` installed (works fine on stable 2017.2.4):

```
Testing started at 14:07 ...
/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/bin/python "/Users/jetbrains/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-C/ch-4/173.3531.8/PyCharm CE 2017.3 EAP.app/Contents/helpers/pydev/pydevd.py" --multiproc --qt-support=auto --client 127.0.0.1 --port 51909 --file "/Users/jetbrains/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-C/ch-4/173.3531.8/PyCharm CE 2017.3 EAP.app/Contents/helpers/pycharm/_jb_pytest_runner.py" --path /Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/test_sample.py
pydev debugger: process 2260 is connecting

Connected to pydev debugger (build 173.3531.8)
Launching py.test with arguments /Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/test_sample.py in /Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin

============================= test session starts ==============================
platform darwin -- Python 3.6.3, pytest-3.2.3, py-1.4.34, pluggy-0.4.0
rootdir: /Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin, inifile:
plugins: pycharm-0.4.0
collected 1 item
test_sample.py F
test_sample.py:0 (test_dummy)
def test_dummy():
        assert True
>       assert False, 'It broke...'
E       AssertionError: It broke...
E       assert False

test_sample.py:3: AssertionError

INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/main.py", line 110, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/main.py", line 146, in _main
INTERNALERROR>     config.hook.pytest_runtestloop(session=session)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 745, in __call__
INTERNALERROR>     return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 339, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 334, in <lambda>
INTERNALERROR>     _MultiCall(methods, kwargs, hook.spec_opts).execute()
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 614, in execute
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/main.py", line 169, in pytest_runtestloop
INTERNALERROR>     item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 745, in __call__
INTERNALERROR>     return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 339, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 334, in <lambda>
INTERNALERROR>     _MultiCall(methods, kwargs, hook.spec_opts).execute()
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 613, in execute
INTERNALERROR>     return _wrapped_call(hook_impl.function(*args), self.execute)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 254, in _wrapped_call
INTERNALERROR>     return call_outcome.get_result()
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 279, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 265, in __init__
INTERNALERROR>     self.result = func()
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 613, in execute
INTERNALERROR>     return _wrapped_call(hook_impl.function(*args), self.execute)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 254, in _wrapped_call
INTERNALERROR>     return call_outcome.get_result()
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 279, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 265, in __init__
INTERNALERROR>     self.result = func()
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 614, in execute
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/runner.py", line 68, in pytest_runtest_protocol
INTERNALERROR>     runtestprotocol(item, nextitem=nextitem)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/runner.py", line 82, in runtestprotocol
INTERNALERROR>     reports.append(call_and_report(item, "call", log))
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/runner.py", line 168, in call_and_report
INTERNALERROR>     hook.pytest_exception_interact(node=item, call=call, report=report)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 745, in __call__
INTERNALERROR>     return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 339, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 334, in <lambda>
INTERNALERROR>     _MultiCall(methods, kwargs, hook.spec_opts).execute()
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/_pytest/vendored_packages/pluggy.py", line 614, in execute
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/jetbrains/PycharmProjects/test_pytest_pycharm_plugin/.env/lib/python3.6/site-packages/pytest_pycharm.py", line 25, in pytest_exception_interact
INTERNALERROR>     debugger = pydevd.debugger
INTERNALERROR> AttributeError: module 'pydevd' has no attribute 'debugger'

=========================== 1 failed in 0.09 seconds ===========================
Process finished with exit code 0

```

--- a/pytest_pycharm.py
+++ b/pytest_pycharm.py
@@ -22,7 +22,10 @@ def pytest_exception_interact(node, call, report):
         frame = frames[-1]
         exception = (exctype, value, traceback)
         thread.additional_info.pydev_message = 'test fail'
-        debugger = pydevd.debugger
+        try:
+            debugger = pydevd.debugger
+        except AttributeError:
+            debugger = pydevd.get_global_debugger()
         pydevd_tracing.SetTrace(None)  # no tracing from here
         debugger.handle_post_mortem_stop(thread, frame, frames_by_id, exception)
 

